src/Files.App/Filesystem/FilesystemOperations/FilesystemOperations.cs

```diff
@@ -887,6 +914,36 @@ public Task<IStorageHistory> CreateShortcutItemsAsync(IList<IStorageItemWithPath
 			throw new NotImplementedException("Cannot create shortcuts in UWP.");
 		}
 
+		private async void RequestPassword(object? sender, TaskCompletionSource<StorageCredential> e)
```

Comment:
I cannot be check myself because I am reviewing on phone, but this method seems equivalent to `private async void RequestPassword(object? sender, TaskCompletionSource<StorageCredential> e)` from `ItemViewModel`. Can it be fused?

Reply:
True, do you know where should I put it?

Reply:
I'll give a look tonight